helm/charts/determined/templates/genai/genai-persistent-volume-claim.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if or (not .Values.genai.version) .Values.genai.sharedPVCName }}
+{{ else }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ include "genai.PVCName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ include "genai.PVCName" . }}
+    release: {{ .Release.Name }}
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ required  "A valid .Values.genai.generatedPVC.storageSize entry is required if a .Values.sharedPVCName is not specified" (and .Values.genai.generatedPVC .Values.genai.generatedPVC.storageSize) }}
+  {{- if .Values.genai.generatedPVC.storageClassName }}
+  storageClassName: {{ .Values.genai.generatedPVC.storageClassName }}
```

Comment:
should we require `storageClassName` parameter as well? I don't think there's any setup when the default can lead to a shared volume being created.